code/modules/mob/living/silicon/silicon.dm

```diff
@@ -29,6 +29,8 @@
 	var/list/access_rights
 	var/obj/item/card/id/idcard
 	var/idcard_type = /obj/item/card/id/synthetic
+	var/is_watching = FALSE
+	var/can_multiz_pb = FALSE
```

Comment:
These vars should be probably moved to mob/living so there aren't double definitions of the same variables.